src/ts/store/store.ts

```diff
@@ -0,0 +1,65 @@
+import type Actions from './actions';
+import type Mutations from './mutations';
+
+type StoreStatus = 'action' | 'mutation' | 'resting';
+
+export interface StoreInterface {
+  readonly commit: <T>(mutationKey: keyof typeof Mutations, payload?: T) => void;
+  readonly dispatch: <T>(actionKey: keyof typeof Actions, payload?: T) => unknown;
+}
+
+export default class implements StoreInterface {
```

Comment:
`class` 명은 일부러 생략하신건가요?

Reply:
넵... 내부에서 호출하는 일이 없기도 하고, default로 export 하는거라 이름을 붙일 필요가 없는 것 같아서요.
사실 이런식으로 할당하는 표현식으로 하고 싶었는데, export default는 이게 안되는 것 같더라구요 ㅜㅜ 
```suggestion
export const store = class implements StoreInterface  {
```